src/javaharness/java/arcs/demo/particles/CaptureKotlinEntity.kt

```diff
@@ -0,0 +1,39 @@
+package arcs.demo.particles
+
+import arcs.api.*
+import arcs.demo.services.ClipboardService
+
```

Comment:
This is just a placeholder automatic conversion of CaptureEntity to Kotlin, it does not use the generated schema2kotlin entity yet, I merely added this to test transitive Kotlin library builds w/arcs_kt_schema

Reply:
Could you add that as a comment to the file (with a TODO to delete it)